source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -1337,6 +1337,10 @@ def _tableHeaderTextHelper(self, axis):
 		cell = self._IATableCell
 		if not cell:
 			return None
+		try:

```

Comment:
There is an IA2UniqueID attribute already cached on the IAccessible NVDAObject at construction time. It will be None if not available.
Thus self.IA2UniqueID can be used directly here rather than queryInterface and fetching again.